ConfigurationSystem/private/Refresher.py

```diff
@@ -143,6 +143,11 @@ def __refresh( self ):
     else:
       lInitialListOfServers = gConfigurationData.getServers()
       gLogger.debug( "Refreshing from list %s" % str( lInitialListOfServers ) )
+
+    # If no servers in the initial list, we are supposed to use the local configuration only
+    if not lInitialListOfServers:
+      return S_OK()
+
```

Comment:
A comment about the naming. It is a mix of "hungarian" and "natural" style. In DIRAC we rely on natural style where the type is the last "camel" word. So, the variable name should be rather:
lInitialListOfServers -> initialServerList
There are several case like that